lib/chewy/index/actions.rb

```diff
@@ -32,7 +32,7 @@ def exists?
         #
         def create(*args, **kwargs)
           create!(*args, **kwargs)
-        rescue Elasticsearch::Transport::Transport::Errors::BadRequest
+        rescue Elastic::Transport::Transport::Errors::BadRequest
```

Comment:
I see that we have very small difference between 7.x and 8.x, can we make it work with both version? For instance as a possibility, define these constants like 
```ruby
if es_8?
  ELASTIC_ERROR_NOT_BAD_REQUEST = Elastic::Transport::Transport::Errors::BadRequest end
else 
  ELASTIC_ERROR_NOT_BAD_REQUEST = Elasticsearch::Transport::Transport::Errors::BadRequest 
end
```
and use it 
```
 def create(*args, **kwargs)
          create!(*args, **kwargs)
        rescue ELASTIC_ERROR_NOT_BAD_REQUEST
```

Or even make this prefix configurable 
```
if es_8?
  ELASTIC_LIB_CONST_PREFIX = Elastic
else 
  ELASTIC_LIB_CONST_PREFIX = Elasticsearch
end
```

and then `ELASTIC_LIB_CONST_PREFIX::Transport::Transport::Errors::BadRequest`, not sure if it's going to work :D